language/ractor_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../spec_helper'
+
+describe "Magic comments" do
```

Comment:
These examples seem to clearly come from CRuby tests.
While that in itself is not a problem, it also doesn't improve coverage.
In general, it's much better to start from scratch when writing specs, to discover edge cases and have a structure of tests testing the various possibilities.

Reply:
I'm a newbie when using Ractors, so I stole the code from CRuby to have something to start and have the foundation for further cases